### PR TITLE
[ko] Update translation

### PR DESCRIPTION
**Pull request by marocchino**
- Some update
- Do not use colon
- Other minor fixes
- Unify words: command line, argument

Review please. @Dalgona @devleoper @riseshia @malkoG

### CODE REVIEW

**Comment by devleoper** [comment]
Looks nice! I thought Unifying terms and other changes you did was huge work!
Thanks for your contribution. :tada: Does anybody have an opinion about that?

File: ko/lessons/advanced/concurrency.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ iex> spawn(Example, :add, [2, 3])
 
 ### 메시지 전달
 
-프로세스끼리 통신할 때 메시지 전달을 이용합니다. 메시지 전달은 주로 두 가지로 나뉘어 집니다: `send/2`랑 `receive`죠. `send/2`함수는 PID에 메시지를 전달할 수 있게 해줍니다. 메시지를 전달받을 때는 `receive`로 메시지를 매칭합니다. 매칭되는 것이 없다면, 인터럽트되지 않은 채로 실행이 계속됩니다.
+프로세스끼리 통신할 때 메시지 전달을 이용합니다. 메시지 전달은 주로 `send/2`, `receive` 두 가지로 나뉘어 집니다. `send/2`함수는 PID에 메시지를 전달할 수 있게 해줍니다. 메시지를 전달받을 때는 `receive`로 메시지를 매칭합니다. 매칭되는 것이 없다면, 인터럽트되지 않은 채로 실행이 계속됩니다.
```
Comment (riseshia) [inline ko/lessons/advanced/concurrency.md]: (maybe) `receive` 두 가지로 나뉘어 집니다. -> `receive`로 나뉘어 집니다.
we may not need `두 가지` XD

File: ko/lessons/advanced/error-handling.md [hunk 2]
```diff
@@ -73,28 +72,28 @@ The end!
...
+   # 위험이 큰 함수를 실행
 after
    File.close(file)
 end
 ```
 
 ## 새로운 에러
 
-Elixir가 `RuntimeError`와 같은 내장 에러 타입을 여럿 포함하고 있지만, 우리는 필요에 따라 그에 맞는 에러를 만들어 낼 수 있어야 합니다. `defexception/1` 매크로를 이용하여 새로운 에러를 쉽게 만들어 낼 수 있습니다. `defexception/1` 매크로는 `:message` 옵션을 인자로 받아, 편리하게 기본 에러 메시지를 설정할 수 있습니다.
+Elixir가 `RuntimeError`와 같은 내장 에러 타입을 여럿 포함하고 있지만, 필요하다면 그에 맞는 에러를 만들수 있어야 합니다. `defexception/1` 매크로를 이용하여 새로운 에러를 쉽게 만들수 있습니다. `defexception/1` 매크로는 `:message` 옵션을 인자로 받아, 편리하게 기본 에러 메시지를 설정할 수 있습니다.
```
Comment (riseshia) [inline ko/lessons/advanced/error-handling.md]: `만들수` -> `만들 수`

File: ko/lessons/basics/testing.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ Finished in 0.03 seconds (0.02s on load, 0.01s on tests)
 
 테스트 코드를 써 보신 적이 있다면 `assert`라는 단어에 익숙할 겁니다. 몇몇 프레임워크에서는 `assert` 대신 `should`나 `expect`를 사용하지만, 그래도 익숙하기는 마찬가지입니다.
 
-해당 조건식이 참인지 테스트할 때 `assert` 매크로를 사용합니다. 참이 아니게 되는 일이 벌어진다면, 테스트가 실패하면서 에러를 던집니다. 테스트가 실패하는 경우를 확인해보기 위해서 샘플 테스트 코드를 수정하고 `mix test`를 실행해봅시다:
+해당 표현식이 참인지 테스트할 때 `assert` 매크로를 사용합니다. 참이 아니라면,  에러를 발생시키며 테스트가 실패합니다. 테스트가 실패하는 경우를 확인해보기 위해서 샘플 테스트 코드를 수정하고 `mix test`를 실행해봅시다.
```
Comment (riseshia) [inline ko/lessons/basics/testing.md]: `  ` two space

**Comment by riseshia** [comment]
I'm done :)

**Pull request merged**